src/widgets/Header.m.css

```diff
@@ -0,0 +1,209 @@
+.root {
+	--toggle-button-height: 20px;
+	--toggle-button-width: 26px;
+	--header-background-color: #212329;
+	--header-link-color: #999;
+	--shadow-color: rgba(0, 0, 0, 0.5);
+	--toggle-button-color: #d3d3d3;
+	--link-color: #fff;
+	position: fixed;
+	width: 100%;
+	background: var(--header-background-color);
+	display: flex;
+	justify-content: flex-start;
+	align-content: flex-start;
+	align-items: center;
+}
+
+.menu {
+}
+
+.menuList {
+	list-style: none;
+	display: flex;
+	padding: 0;
+	height: auto;
+	width: auto;
+	background: none;
+}
+
+.menuItem {
+	text-align: right;
+	margin: 10px;
+}
+
+.left {
+	display: flex;
+	align-items: center;
+	flex: 1 0 auto;
+}
+
+.link {
+	font-size: 0.9rem;
+	min-width: 140px;
+	text-align: center;
+	text-decoration: none;
+	color: var(--link-color);
+	box-sizing: border-box;
+	color: var(--header-link-color);
+	text-transform: uppercase;
+	transition: 0.1s ease-in-out;
+	transition-property: color, background-color;
+}
+
+.homeLink {
+}
+
+.selected {
+}
+
+.logoContainer {
+}
+
+.logo {
+	height: 48px;
+	width: 110px;
+}
+
+.backdrop {
+}
+
+.expanded .backdrop {
+	position: fixed;
+	display: block;
+	content: '';
+	left: 0;
+	top: 0;
+	width: 100%;
+	height: 100%;
+	z-index: 997;
+	background: var(--shadow-color);
+	cursor: default;
+}
+
+.toggleButtonContainer {
+	margin: auto 10px;
+}
+
+.toggleButton {
+	text-align: left;
+	background: none;
+	font: inherit;
+	position: relative;
+	cursor: pointer;
+	height: var(--toggle-button-height);
+	width: var(--toggle-button-width);
+	z-index: 2;
+	top: calc(50 - var(--toggle-button-height));
+	border: none;
+	display: inline-block;
+	padding: 0;
+	margin: 0;
+}
+
+.toggleBar {
+	margin-top: 0;
+}
+
+.toggleBar::after {
+	margin-top: calc(var(--toggle-button-height) / 2.5);
+}
+
+.toggleBar::before {
+	margin-top: calc(calc(var(--toggle-button-height) / 2.5) * -1);
+}
+
+.toggleBar,
+.toggleBar::after,
+.toggleBar::before {
+	position: absolute;
+	top: 50%;
+	transform: translateY(-50%);
+	transition: all 0.5s ease;
+	background: var(--toggle-button-color);
+	border-radius: 3px;
+	content: '';
+	height: calc(var(--toggle-button-height) / 5);
+	width: 100%;
+}
+
+.srOnly {
```

Comment:
Is it worth commenting this? I can't remember what this does but I feel like you explained it last PR

Reply:
Commented!